lib/parser/hiredis.js

```diff
@@ -27,12 +27,7 @@ HiredisReplyParser.prototype.execute = function (data) {
     var reply;
     this.reader.feed(data);
     while (true) {
-        try {
-            reply = this.reader.get();
-        } catch (err) {
-            this.emit("error", err);
-            break;
-        }
+        reply = this.reader.get();
```

Comment:
It is safe to remove this try catch as there's a try catch calling the execute function and emitting the error just as it was done here before (besides the fact that this code block was never reached in our tests).